src/frontend/components/modal/lib/component.js

```diff
@@ -271,6 +271,7 @@ class ModalComponent extends Component {
 
   // Validate the required fields of the frame
   validateFrame() {
+    if(!this.frame) return;
```

Comment:
This is an "odd" bug that's just arrived - I don't know if it was present before, but if the frame hasn't been visible, it is set to undefined/null, therefore setting invalid was throwing an exception.